providers/aws/provider/provider.go

```diff
@@ -203,15 +203,6 @@ func (s *Service) connect(req *plugin.ConnectReq, callback plugin.ProviderCallba
 
 		case string(awsec2ebsconn.EBSConnectionType):
 			conn, err = awsec2ebsconn.NewAwsEbsConnection(connId, conf, asset)
-			if conn.Asset() != nil && len(conn.Asset().Connections) > 0 && conn.Asset().Connections[0].Options["mounted"] != "" {
```

Comment:
im not sure why this was needed. the ebs conn seems to work without it just fine

Reply:
built in. i can try testing with the provider binary separate too

Reply:
Just tested, no issues with a non-built in provider